src/osd/ECBackend.h

```diff
@@ -136,20 +136,29 @@ class ECBackend : public PGBackend {
    * ensures that we won't ever have to restart a client initiated read in
    * check_recovery_sources.
    */
+  typedef boost::tuple<uint64_t, uint64_t, uint32_t> ec_align_t;
+  typedef std::map<hobject_t,std::pair<int, extent_map>>  ec_extents_t;
```

Comment:
using, not typedef

Reply:
Swapping these types for their typedefs makes this PR harder to read.  If we're going to bother with this, at least ec_align_t should be a struct with useful member names.  Those changes should also be their own commit.